architecture/core_concepts/routes.adoc

```diff
@@ -196,17 +196,17 @@ variables on the *deployment config* for the router to alter its configuration.
 |`*DEFAULT_CERTIFICATE*` |  | The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format.
 |`*DEFAULT_CERTIFICATE_DIR*` |  | A path to a directory that contains a file named tls.crt. If tls.crt is not a PEM file which also contains a private key, it is first combined with a file named tls.key in the same directory. The PEM-format contents are then used as the default certificate. Only used if DEFAULT_CERTIFICATE or DEFAULT_CERTIFICATE_PATH are not specified.
 |`*DEFAULT_CERTIFICATE_PATH*` |  | A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format. Only used if DEFAULT_CERTIFICATE is not specified.
-|`*EXTENDED_VALIDATION*` | true | Whether to perform an additional extended validation step on all routes admitted by this router.
+|`*EXTENDED_VALIDATION*` | true | If __"true"__,perform an additional extended validation step on all routes admitted by this router.
 |`*NAMESPACE_LABELS*` |  | A label selector to apply to namespaces to watch, empty means all.
 |`*PROJECT_LABELS*` |  | A label selector to apply to projects to watch, emtpy means all.
 |`*RELOAD_SCRIPT*` |  | The path to the reload script to use to reload the router.
-|`*ROUTER_BACKEND_CHECK_INTERVAL*` | 5000ms | Time interval between subsequent "liveness" checks on backends.
+|`*ROUTER_BACKEND_CHECK_INTERVAL*` | 5000ms | Time interval between subsequent __"liveness"__ checks on backends.
```

Comment:
Something like "Length of time between..." might be more accurate.

This would apply to a few of the other sentences in this section as well.